client/src/components/Allocations/AllocationsModals/AllocationsManageTeamTable/AllocationsManageTeamTable.module.scss

```diff
@@ -0,0 +1,72 @@
+.listing-wrapper {
+  height: 33.5vh;
+  overflow-y: auto;
+  padding-left: 0;
+  padding-right: 0;
+}
+.manage-team-table {
+  font-size: 0.875rem; /* 14px (approved deviation from design) */
+  margin-bottom: 0; /* overwrite bootstrap */
+  border-collapse: separate; /* keep <th> border-bottom visible on scroll */
+  border-spacing: 0;
+
+  td,
+  th {
+    padding: 9px 15px;
+    vertical-align: middle;
+  }
+  th {
+    font-weight: bold;
+    background-color: var(--global-color-primary--xx-light);
+  }
+  thead th /* specificity overwrites bootstrap */ {
+    border-bottom: 1px solid var(--global-color-primary--dark);
+  }
+  /* action */
+  tr > *:nth-child(3) {
+    text-align: right;
+    padding-right: 40px;
+
+    /* To not let button appear atop header on scroll */
+    button {
+      position: static; /* overwrite <Button> */
+    }
+  }
+  tr:nth-child(even) td {
+    background-color: var(--global-color-primary--x-light);
+
+    button:disabled {
+      background-color: var(
+        --global-color-primary--x-light
+      ); /* button's disabled state background will match row background color */
+    }
```

Comment:
Let's fix this everywhere, not just for Allocations:
- https://github.com/TACC/Core-Portal/pull/659